<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDeviceInfo</class>
 <widget class="QWidget" name="FormDeviceInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget
{
	background-color: rgba(255, 255, 255, 0);
	border-bottom:1px,rgb(255,255,255),solid;
	selection-border-image: url(:/png/logined_selected_row_bg.png);
	
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	font-size: 16px;
	font-family:Microsoft Yahei;

}

</string>
   </property>
   <widget class="QLabel" name="labelType">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>231</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelIP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>211</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>131</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFactory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>厂商</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
